src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final FigureName figureName = FigureName.CIRCLE;
```

Comment:
The `figureName` field should not be static. Each instance of `Circle` will have its own name, and that name should not be shared across all instances.